Database/input-data/pmount/13c0774012192e42c5fe203be7ec887869a883dc93f2c76d77777ac026e2f1e3.json

```diff
@@ -0,0 +1,44 @@
+{
+    "Conventions":"CF-1.7",
+    "activity_id":"input4MIPs",
+    "comment":"LUH3 V0",
+    "comment_post_publication":null,
+    "contact":"lchini@umd.edu, gchurtt@umd.edu",
+    "creation_date":"2025-03-10T16:15:14Z",
+    "data_node":null,
+    "dataset_category":"landState",
+    "datetime_end":"2024-01-01T00:00:00Z",
+    "datetime_start":"0850-01-01T00:00:00Z",
+    "doi":"10.25981/ESGF.input4MIPs.CMIP7/2521499",
+    "esgf_dataset_master_id":"input4MIPs.CMIP7.CMIP.UofMD.UofMD-landState-3-1.land.yr.multiple-management.gn.v20250310",
+    "filepath":"/p/user_pub/work/input4MIPs/CMIP7/CMIP/UofMD/UofMD-landState-3-1/land/yr/multiple-management/gn/v20250310/multiple-management_input4MIPs_landState_CMIP_UofMD-landState-3-1_gn_0850-2024.nc",
+    "frequency":"yr",
+    "further_info_url":"http://luh.umd.edu",
+    "grid":null,
+    "grid_label":"gn",
+    "institution":"University of Maryland (UofMD), College Park, MD 20742, USA",
+    "institution_id":"UofMD",
+    "latest":null,
+    "license":"Land-Use Harmonization data produced by the University of Maryland is licensed under a Creative Commons Attribution \\\"Share Alike\\\" 4.0 International License (http://creativecommons.org/licenses/by/4.0/). The data producers and data providers make no warranty, either express or implied, including but not limited to, warranties of merchantability and fitness for a particular purpose. All liabilities arising from the supply of the information (including any liability arising in negligence) are excluded to the fullest extent permitted by law.",
+    "license_id":null,
+    "mip_era":"CMIP7",
+    "nominal_resolution":"25 km",
+    "product":null,
+    "publication_status":"in_publishing_queue",
+    "realm":"land",
+    "references":"Hurtt et al. 2020 (https://doi.org/10.5194/gmd-13-5425-2020), Chini et al. 2021 (https://doi.org/10.5194/essd-13-4175-2021)",
+    "region":null,
+    "replica":null,
+    "sha256":"13c0774012192e42c5fe203be7ec887869a883dc93f2c76d77777ac026e2f1e3",
+    "source":"LUH3 V0: Land-Use Harmonization Data Set for CMIP7",
```

Comment:
Ditto here - LUH3 V1 - inconsistencies across files

Reply:
Too late, data has been published. Added to #215